charts/merlin/values.yaml

```diff
@@ -0,0 +1,463 @@
+deployment:
+  image:
+    pullPolicy: IfNotPresent
+    registry: ghcr.io
+    repository: gojek/merlin
+    tag: 0.21.0
```

Comment:
I just tested, this image is not using the new google identity service lib, so after logging in using MLP ui, Merlin's UI cannot be accessed. We should switch to at least v0.23.0-rc1

Reply:
updated